code/modules/clothing/suits/labcoat.dm

```diff
@@ -45,43 +45,51 @@
 	desc = "Bluer than the standard model."
 	base_icon_state = "labcoat_cmo"
 	item_state = "labcoat_cmo"
+	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 70, rad = 0)
 	species_fit = list(VOX_SHAPED, GREY_SHAPED)
 
 /obj/item/clothing/suit/storage/labcoat/mad
-	name = "The Mad's labcoat"
+	name = "the Mad's labcoat"
```

Comment:
This one should stay, "The Mad" is a person and with your current change it'll read as the the Mad's labcoat.